consensus/abci.go

```diff
@@ -65,18 +64,14 @@ func (abci *TendermintABCI) InitEthState() error {
 
 // Info returns information about the last height and app_hash to the tmtCfg engine
 func (abci *TendermintABCI) Info(req tmtAbciTypes.RequestInfo) tmtAbciTypes.ResponseInfo {
-	abci.logger.Info("TendermintABCI::Info()", "data", req)
```

Comment:
Restore as `Debug()`

Reply:
I added a better logging, is not removed